subcommands/targets/show.go

```diff
@@ -238,17 +240,32 @@ func doShowSboms(cmd *cobra.Command, args []string) {
 	logrus.Debugf("Showing sboms for %s %s", factory, version)
 
 	prodTag, _ := cmd.Flags().GetString("production-tag")
+	downloadPath, _ := cmd.Flags().GetString("download")
 
 	_, _, targets := getTargets(factory, prodTag, version)
 	for name := range targets {
 		sboms, err := api.TargetSboms(factory, name)
 		subcommands.DieNotNil(err)
-		t := tabby.New()
-		t.AddHeader("BUILD", "ARTIFACT")
-		for _, sbom := range sboms {
-			t.AddLine(sbom.CiBuild+"/"+sbom.CiRun, sbom.Artifact)
+
+		if len(downloadPath) > 0 {
+			st, err := os.Stat(downloadPath)
+			subcommands.DieNotNil(err)
```

Comment:
Below we assume that it is a directory; so I would check for that here explicitly.
Also, it might make sense to check that directory is empty (so that we don't accidentally overwrite its contents).